hbase-client/src/main/java/org/apache/hadoop/hbase/client/AsyncConnection.java

```diff
@@ -34,6 +34,16 @@
 @InterfaceAudience.Public
 public interface AsyncConnection extends Closeable {
 
+  /**
+   * Returns the clusterId of this connection.
+   */
+  String getClusterId2();
```

Comment:
This is not a good method name for a public API. Please find a better one.

Reply:
Updated.